content/providers/01-ai-sdk-providers/03-openai.mdx

```diff
@@ -224,6 +224,9 @@ The following provider options are available:
 - **promptCacheKey** _string_
   A cache key for manual prompt caching control. Used by OpenAI to cache responses for similar requests to optimize your cache hit rates.
 
+- **promptCacheRetention** _'in_memory' | '24h'_
+  The retention policy for the prompt cache. Set to `'24h'` to enable extended prompt caching, which keeps cached prefixes active for up to 24 hours. Defaults to `'in_memory'` for standard prompt caching. Note: `'24h'` is currently only available for the 5.1 series of models.
```

Comment:
I'm afraid that the mention of 5.1 series could get outdated soon and we will miss it.

Maybe rephrase to "is not supported by all models" or is not supported by models prior the 5.1 series?

@nicoalbanese